### DISCUSSION TITLE
👋 Welcome!

### DISCUSSION THREAD

**Discussion by owenlongbo** [illacloud/illa-builder#24] [Announcements]
⭐ ILLA is an open-source low-code platform. 
 Github Link: [https://github.com/illa-family](https://github.com/illa-family)
 Discord Link: [https://discord.gg/2tGBuJkgd6](https://discord.gg/2tGBuJkgd6) 

## ✨We want to do
 Help developers build business tools more efficiently.

## 🌟Let's Code Together
 **This is ILLA Official Community.**

  We're using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you're wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thanhhiepvo** [comment]
It similar to no-code than low-code

**Comment by dasaf7** [comment]
Unfortunately, Discord link doesn't work.

**Comment by whenever** [comment]
I understand that IllaCloud is being discontinued, is there a community forum or chat group that is actively discussing Illa?
